src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface TodListProps {
```

Comment:
There is a typo in the interface name `TodListProps`. It should be `TodoListProps` to match the component name and maintain consistency.